CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 ### Changed
+- Improve how includes with relative paths are resolved during parsing/compilation.
```

Comment:
Something like this explains the change in a better way.

```suggestion
- Resolve include paths relative to the including file path when the resolution relative to the root file path fails.
```

Reply:
Reformulated.